### PR TITLE
RTCQuicTransport Constructor Checks (and internal slots)

### PR DESCRIPTION
**Pull request by aboba**
Fix for Issue https://github.com/w3c/webrtc-quic/issues/31

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -201,15 +214,15 @@ <h2>Attributes</h2>
             <dt><dfn><code>state</code></dfn> of type <span class=
             "idlAttrType"><a>RTCQuicTransportState</a></span>, readonly</dt>
             <dd>
-              <p>The current state of the QUIC transport.</p>
+              <p>The current state of the QUIC transport. On getting, it MUST return
+              the value of the <a>[\[QuicTransportState]]</a> internal slot.</p>
```
Comment (shampson) [inline index.html]: [\[QuicTransportState]] --> [[\QuicTransportState]]?

**Comment by shampson** [comment]
Just that small fix, otherwise looks good!

**Comment by shampson** [comment]
lgtm. Should I go ahead and merge?

File: index.html (hunk 1 above)
Comment (aboba) [reply index.html]: Fixed.

**Pull request merged**